Proposal (#2)
twotango/research: proposal/index.md

# Research proposal

> I propose the following research project: 

Reviewer: ```suggestion
```

> Given the availability of the ASL database as a global sign language dataset and the need to customize personal datasets for sufficient guessability and high subjective ratings of user-defined gesture sets to produce suitable, context-dependent sentences, how can the design of multimodal input interfaces be optimized to support seamless transitions between different input modes, including tap, motion, and image gestures, and enhance the overall user experience?

Reviewer: ```suggestion
Given the availability of the [ASL database](https://asl-lex.org) as a global sign language dataset, the Neurodiversity Foundation needs a customized personal datasets. We intend to used this for sufficient guessability and high subjective ratings of user-defined gesture sets. We want to produce suitable, context-dependent sentences. Based on this context, we would like to know how we can design a multimodal input interfaces that can be optimized to support seamless transitions between different the following gesture input modes: tap gestures, device motion gestures (via accelerometer and gyroscope data), hand motion gestures (via image/video recognition). The goal is to enhance upon the current standards of keyboard accessibility on devices, by enabling more non-verbal inputs.
```

> For related questions refer to questions.md

Reviewer 2: ```suggestion
For related questions refer to the [questions](questions.md) file.
```
